tests/fc/run_fullyconnected.sh

```diff
@@ -98,13 +98,14 @@ if [ "" = "${LIBXSMM_TARGET_HIDDEN}" ] || [ "0" = "${LIBXSMM_TARGET_HIDDEN}" ];
 fi
 
 if [ "f32" == "${BIN}" ]; then
-  PREC_BF16=0
```

Comment:
See comment on string "F32, Bf16, BF8" in the driver as this prepares better for HF8